hack/xref-helpmsgs-manpages

```diff
@@ -95,6 +95,13 @@ my %Format_Option_Is_Special = map { $_ => 1 } (
     'inspect',                                  # ambiguous (container/image)
  );
 
+# Hardcoded list of existing duplicate-except-for-case format codes,
+# with their associated subcommands. Let's not add any more.
+my %Format_Option_Dup_Allowed = (
+    'podman-images' => { '.id'     => 1 },
+    'podman-stats'  => { '.avgcpu' => 1, '.pids' => 1 },
```

Comment:
I do not see them duplicated in the output of the shell completion (`podman __complete stats --format {{.`)?
Should not not just remove the the dups in the man page?

Reply:
It's easier to see the duplication if you `| sort`.

Reply:
P.S. "remove the dups in the man page" is impossible, because of course the `xref` script compares completion against man page. If you removed something from the man page, the script would fail.

Reply:
I could do a case-sensitive comparison but then we would allow introducing more of these, and I strongly consider these to be bugs. It is evil to have `.Id` and `.ID`. We should not permit that to happen again.